requirements.txt

```diff
@@ -1,4 +1,4 @@
-mlflow>=2.0
```

Comment:
Is there a reason we removed the mlflow requirement?

Reply:
Thanks, good catch! Added it back.